db/test/test_dataset_eval.py

```diff
@@ -162,3 +162,8 @@ def test_eval_job_location(self):
         job2_id = dataset_eval._create_job(self.conn, self.test_dataset_id, True, dataset_eval.EVAL_LOCAL, None)
         job2 = dataset_eval.get_job(job2_id)
         self.assertEqual(job2["eval_location"], dataset_eval.EVAL_LOCAL)
+
+    def test_get_pending_jobs_for_user(self):
+        job_id = dataset_eval._create_job(self.conn, self.test_dataset_id, True, dataset_eval.EVAL_REMOTE, None)
```

Comment:
Sure - there are many statuses that a job can have. I recommend you have one in pending, and one in not pending - it doesn't matter which one.